configs/kinobi.cjs

```diff
@@ -87,6 +87,21 @@ kinobi.update(
         return k.structFieldTypeNode({ ...node, name: "metadata" });
       },
     },
+    {
+      // Update `collectionAuthorityRecordPda` account as `optional`.
+      selector: {
+        kind: "instructionAccountNode",
+        name: "collectionAuthorityRecordPda",
+      },
+      transformer: (node) => {
+        k.assertInstructionAccountNode(node);
+        return k.instructionAccountNode({
+          ...node,
+          isOptional: true,
+          isOptionalStrategy: "programId",
```

Comment:
From Kinobi 0.14, this was moved to the instruction node instead.

https://github.com/metaplex-foundation/kinobi/blob/main/src/nodes/InstructionNode.ts#L44

So you don't need to include this line at all.

```suggestion
```